addon/plugins/color/marks/color.ts

```diff
@@ -0,0 +1,22 @@
+import { Mark, MarkSpec } from 'prosemirror-model';
+
+export const color: MarkSpec = {
+  attrs: {
+    color: {},
+  },
+  parseDOM: [
+    {
+      style: 'color',
+      getAttrs: (value) => {
+        return {
+          color: value,
+        };
+      },
+    },
+  ],
+  toDOM(mark: Mark) {
+    const { color } = mark.attrs;
+
+    return ['span', { style: `color: ${color as string}`, mark: 'color' }, 0];
```

Comment:
I think it would be better to use a `data-mark` attribute instead of `mark`, so it is legal.

Reply:
Done